src/microbe_stage/SpawnSystem.cs

```diff
@@ -157,12 +157,12 @@ public void Process(float delta, Vector3 playerPosition)
         // Remove the y-position from player position
         playerPosition.y = 0;
 
-        int spawnsLeftThisFrame = Constants.MAX_SPAWNS_PER_FRAME;
+        float spawnsLeftThisFrame = Constants.MAX_SPAWNS_PER_FRAME;
 
         // If we have queued spawns to do spawn those
         HandleQueuedSpawns(ref spawnsLeftThisFrame, playerPosition);
 
-        if (spawnsLeftThisFrame <= 0)
+        if (spawnsLeftThisFrame <= 0.0f)
```

Comment:
```suggestion
        if (spawnsLeftThisFrame <= 0)
```
This compiles, right? I prefer code like this to check that a float is not negative.